pipeline_steps/common.groovy

```diff
@@ -598,4 +591,24 @@ void override_inventory(){
   )
 }
 
+// initialisation steps for nodes
+void use_node(label=null, body){
+  node(label){
+    deleteDir()
+    install_ansible()
```

Comment:
install_ansible() assumes that rpc-gating has already been cloned since the repo for requirements/constraints/role_requirements.